apps/graphql/src/apps/itinerary/helpers/Itineraries.js

```diff
@@ -37,10 +38,19 @@ export const mapVehicle = (type: ?string, uniqueNo: ?string) => ({
   uniqueNo: uniqueNo ?? null,
 });
 
-export const sanitizeCarrier = (routeItem: ?ApiRouteItem) => ({
-  name: null, // @TODO - value is currently missing from API endpoint response
-  code: routeItem?.airline ?? null,
-});
+export const sanitizeCarrier = (routeItem: ?ApiRouteItem) => {
+  // @TODO - value is currently missing from API endpoint response
+  // JSON file is used from https://github.com/BesrourMS/Airlines/blob/master/airlines.json
+  const iata = routeItem?.airline;
```

Comment:
iata seems a little bit obscure especially for me that didn't know what it was! we could just name it `airlineCode` or something like that and then 
```
return {
    name: name ?? null,
    code: airlineCode ?? null,
  };
```

Reply:
Alright, changed to airlineCode